app/assets/javascripts/manager_custom.coffee

```diff
@@ -1,3 +1,6 @@
 # $(window).load ->
 #   $('.product-images').slick dots: true
 #   return
+$(window).load ->
```

Comment:
ready = ->
# some coffee code here


# in the end of file insert this lines
$(document).ready(ready)
$(document).on('page:load', ready)